src/applications/claims-status/components/claim-status-tab/RecentActivity.jsx

```diff
@@ -234,22 +254,42 @@ export default function RecentActivity({ claim }) {
                 </>
               )}
 
-              {item.status === 'NEEDED_FROM_OTHERS' && (
+              {item.status === 'NEEDED_FROM_OTHERS' &&
+              cstFriendlyEvidenceRequests &&
+              item.activityDescription ? (
                 <va-alert
                   class="optional-alert vads-u-padding-bottom--1"
                   status="info"
                   slim
                 >
-                  You don’t have to do anything, but if you have this
-                  information you can{' '}
+                  {item.activityDescription}
+                  <br />
                   <Link
                     aria-label={`Add information for ${item.displayName}`}
                     className="add-your-claims-link"
                     to={`../document-request/${item.id}`}
                   >
-                    add it here.
+                    About this notice
                   </Link>
                 </va-alert>
+              ) : (
+                item.status === 'NEEDED_FROM_OTHERS' && (
+                  <va-alert
+                    class="optional-alert vads-u-padding-bottom--1"
+                    status="info"
+                    slim
+                  >
+                    You don’t have to do anything, but if you have this
+                    information you can{' '}
+                    <Link
+                      aria-label={`Add information for ${item.displayName}`}
+                      className="add-your-claims-link"
+                      to={`../document-request/${item.id}`}
+                    >
+                      add it here.
```

Comment:
This accessible label also needs to include the visible text "About this notice", or change the visible text to "Add information". Right now the visible label is ignored by screen readers for the `aria-label` which was a bit confusing and Voice Access can't click the link at all because of the difference.